### PR TITLE
doc: disable all tags building in versioning

### PR DESCRIPTION
**Pull request by jd**
There is little sense in building docs for all tags. The stable branches ought
to be enough anyway. Let's skip them entirely.

### CODE REVIEW

**Comment by jd** [comment]
Closing so it does not get merged. @sileht the banner is wrong with this patch and I'm not sure how to fix it. It says on stable/* that this is the dev version and that "master" should be checked. Any hint?

**Comment by jd** [comment]
Reopening and dismissing reviews, simpler :)

File: doc/source/conf.py [hunk 1]
```diff
@@ -183,7 +183,7 @@
 scv_banner_greatest_tag = True
```
Comment (sileht) [inline doc/source/conf.py]: I think we should replace:

scv_banner_greatest_tag = True

by 

scv_banner_main_ref = "master"

Comment (jd) [inline doc/source/conf.py]: That does not solve it because it does *not* show the banner in master. That's where it should be. :/

Comment (sileht) [inline doc/source/conf.py]: So we need:

-scv_banner_greatest_tag = True
+scv_banner_main_ref = 'stable/4.0'

Comment (jd) [inline doc/source/conf.py]: Ok, we need to find a way so that's dynamically computed later then… :)

**Comment by jd** [comment]
Pull-request updated, HEAD is now 5f3c70ad1b56949f5af248fab8c9f71d442afe6e

**Comment by jd** [comment]
Pull-request updated, HEAD is now 49196a37b0062180ec71c4b7444eee3277f59bcc

**Pull request merged**